app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -231,25 +235,44 @@ export default function ManageChannelMembers({
 
     useNavButtonPressed(MANAGE_BUTTON, componentId, toggleManageEnabled, [toggleManageEnabled]);
 
-    useEffect(() => {
-        mounted.current = true;
-        const options: GetUsersOptions = {sort: 'admin', active: true, per_page: PER_PAGE_DEFAULT};
+    const handleReachedBottom = useCallback(() => {
+        if (hasMoreProfiles.current && !loading && !searchedTerm) {
+            pageRef.current += 1;
+            setLoading(true);
+            getFetchChannelMembers();
+        }
+    }, [loading, searchedTerm]);
+
+    const getFetchChannelMembers = useCallback(() => {
+        const options: GetUsersOptions = {sort: 'admin', active: true, per_page: PER_PAGE_DEFAULT, page: pageRef.current};
         fetchChannelMemberships(serverUrl, channelId, options, true).then(({users, members}) => {
```

Comment:
Let's use the `async await` pattern instead of the `promise.then`, so we can avoid the max-nested-callbacks a few lines later.